wolfcrypt/test/test.c

```diff
@@ -878,7 +882,7 @@ wc_test_ret_t wolfcrypt_test(void* args)
 #endif
 
     printf("------------------------------------------------------------------------------\n");
-    printf(" wolfSSL version %s\n", LIBWOLFSSL_VERSION_STRING);
+    printf(" wolfSSL version %s with DevID:%X\n", LIBWOLFSSL_VERSION_STRING,devId);
```

Comment:
Might be confusing to see DevId when its -2 INVALID_DEVID. Can this be conditional?

Reply:
Updated.